participants_logbook.md

```diff
@@ -54,3 +54,5 @@ I really felt immersed and involved with the subjects thanks to all the resource
 * This was my first experience with a school in data science and it was a wonderful one. I learnt a lot and the organization was great. Thank you all for that effort and everything you taught us. Greetings from Peru. Jaime Hoefken
 * Thank you for organizing this summer school. I learned a lot and I look forward to meet and code with our colleagues soon. Thank you for making Astronomy and Data Science more accessible. :telescope: :computer: All the best! :muscle: - Julie Ann Delda
 
+* This summer school was a new experience for me, but definitely a welcome one. The contents will be very helpful for my master's thesis and I'm sure that I will revisit some of the lectures sooner or later. 
+* I hope I can participate in similar events in person soon. Thank you all very much! Martin Schneider 
```

Comment:
Hi
I don't think you want to make a new bullet point here, no?

Reply:
Yes, thanks for letting me know